src/Function/Bundles.agda

```diff
@@ -526,6 +526,16 @@ module _ {A : Set a} {B : Set b} where
     , strictlyInverseʳ⇒inverseʳ to invʳ
     )
 
+------------------------------------------------------------------------
+-- Inline bbreviations for oft-used items
+--  (but one define the ones that recur)
```

Comment:
Typo, for `abreviations`, and the comment in brackets doesn't make sense to me. Also double space at the beginning. Should also read `Infix` not `Inline`.